layouts/index.html

```diff
@@ -1,633 +1,257 @@
-                      repositories</a></li>
-                  <li><a class="link" href="/docker-hub/builds/">Set up automated builds</a>
-                  </li>
-                  <li><a class="link" href="/docker-hub/access-tokens/">Manage access tokens</a>
-                  </li>
-                  <li><a class="link" href="/docker-hub/webhooks/">Docker Hub webhooks</a></li>
-                </ul>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div class="flex gap-4 mx-auto w-[1200px] lg:w-full p-6 sm:flex-col">
-      <div class="w-full">
-        <h2 class="text-3xl">Best practices</h2>
-        <ul class="w-full space-y-3">
-          <li>
-            <a class="link" href="/develop/develop-images/dockerfile_best-practices/">Best
-              practices for writing a Dockerfile</a>
-          </li>
-          <li>
-            <a class="link" href="/develop/develop-images/dockerfile_best-practices/">Best
-              practices for writing a Docker Compose file</a>
-          </li>
-          <li>
-            <a class="link" href="/develop/dev-best-practices/">Docker development best
-              practices</a>
-          </li>
-          <li>
-            <a class="link" href="/engine/security/">Docker security best practices</a>
-          </li>
-        </ul>
-      </div>
-      <div class="w-full">
-        <h2 class="text-3xl">Develop with Docker</h2>
-        <ul class="w-full space-y-3">
-          <li>
-            <a class="link" href="/language/nodejs/">Containerize a Node.js app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/language/python/">Containerize a Python app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/language/java/">Containerize a Java app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/language/golang/">Containerize a Go app using Docker</a>
-          </li>
-          <li>
-            <a class="link" href="/samples/">View more languages and frameworks in Docker
-              samples</a>
-          </li>
-        </ul>
-      </div>
-    </div>
-    <div class="flex mx-auto w-[1200px] flex-col p-6 lg:w-full">
-      <h2 class="text-3xl">Play with Docker</h2>
-      <div class="grid grid-cols-3 gap-4 p-6 sm:flex sm:flex-col xl:grid-cols-2">
-        <div>
-          <a class="h-full" href="/get-started/">
-            <div
-              class="flex h-full flex-col gap-2 rounded border border-gray-light-100 bg-white p-4 drop-shadow-sm hover:border-gray-light-200 hover:drop-shadow-lg dark:border-gray-dark-400 dark:bg-gray-dark-200 hover:dark:border-gray-dark">
-              <div class="flex items-center gap-4">
-                <div>
-                  <div class="text-xl leading-snug text-gray-light-800 dark:text-white">
-                    Tutorial
+            <a class="h-full" href="/desktop/">
```

Comment:
suggestion: i'd find it much more helpful to understand what manuals are and what manuals are available if this linked to `/manuals` instead of dropping me into the docker desktop manual.

Reply:
Yeah this was created before /manuals existed, I'll update